linux_os/guide/system/software/sap/accounts_authorized_local_users_sidadm_orasid.rule

```diff
@@ -0,0 +1,49 @@
+documentation_complete: true
+
+prodtype: ol7
+
+title: 'Only sidadm and orasid/oracle User Accounts Exist on Operating System'
+
+description: |-
+    SAP tends to use the server or virtual machine exclusively. There should be only
+    SAP system users <tt>sidadm</tt> and <tt>orasid</tt> that exist on the operating
+    system (or virtual machine). If SAP Host Agent is installed, the user <tt>sapadm</tt>
+    must exist too. With Oracle Database using <tt>oracle</tt> user, the user <tt>oracle</tt>
+    should exist as well. 
+    While <tt>SID</tt> is the SAP System ID, which is always three alphanumeric characters
+    in upper case, beginning with an alphabetic character, the user names <tt>sidadm</tt> 
+    and <tt>orasid</tt> are in lower case.
+    Besides the above SAP users that are automatically detected, other operating system
+    users can be customized in the refine value variable
+    <tt>var_accounts_authorized_local_users_regex</tt>.
+    OVAL regular expression is used for the user list.
+    <tt>Limitation: Currently this OVAL test only works when there is exactly one SAP
```

Comment:
I feel it would make the prose more readable, if you put the Limitations part into `warning` style. See how to use it https://github.com/OpenSCAP/scap-security-guide/blob/master/linux_os/guide/services/ssh/ssh_server/sshd_disable_rhosts_rsa.rule